packages/gatsby/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "declaration": true,
```

Comment:
I think declaration can be false since we expect all builders to conform to same shape and not export anything else

Reply:
Sounds good.